recipes-firmware/firmware/firmware-qcom-boot-common.inc

```diff
@@ -0,0 +1,20 @@
+# Install NHLOS boot binaries in DEPLOY_DIR
+
+do_install() {
+    rm -f ${S}/${BOOTBINARIES}/partition.xml
```

Comment:
is that really needed?

Reply:
We want to provide a separate recipe for installing partition.xml to accommodate users, who wish to modify the partitions to meet their specific needs.

Reply:
you still don't need to remove this file here. I think you can skip do_install completely, no?

Reply:
yeah, and deploy is not really using any .xml file, so it can probably be skipped.